_assets/stylesheets/base/_variables.scss

```diff
@@ -0,0 +1,4 @@
+/* Variables */
```

Comment:
@tajchumber just a heads up, we have a file for variables located at `_assets/stylesheets/libs/_vars.scss` which is imported on line 1 of `_assets/stylesheets/main.scss`. This file is used for common variables referenced in multiple elements.

Since these variables are presumably only used in `_assets/stylesheets/layout/_sidebar.scss`, I'm going suggest that we stick to the established pattern, and move these variables to line 34, where `$pad` used to be.

Reply:
i was looking for a `_vars.scss` file and was surprised when I couldn't find it. I just didn't look well enough! thanks!